src/actions/emojis.js

```diff
@@ -53,7 +54,7 @@ export function getAllCustomEmojis(perPage = General.PAGE_SIZE_MAXIMUM) {
                     emojis = await Client.getCustomEmojis();
                     hasMore = false;
                 } else {
-                    emojis = await Client4.getCustomEmojis(page, perPage);
+                    emojis = await Client4.getCustomEmojis(page, perPage, 'name');
```

Comment:
what does 'name' mean? are there any other possible options?

Reply:
Sorting by name, and right now it's the only option other than having no sorting

Reply:
Well lets make that a constant if possible